lib/em/pool.rb

```diff
@@ -1,41 +1,41 @@
 module EventMachine
-  # = EventMachine::Pool
```

Comment:
Thank you for checking.
It seems style of Rdoc.
When I executed `bundle exec rake yard`, `doc/EventMachine/Pool.html` was generated.